__tests__/app.test.js

```diff
@@ -40,13 +40,46 @@ describe("GET /api", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id", () => {
+  test("200:should respond with an individual article", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toHaveProperty("article_id", expect.any(Number));
```

Comment:
Here we can strictly assert that the id will be 1, instead of expecting any random number 👍